demos/ota/common/include/mqtt_subscription_manager.h

```diff
@@ -0,0 +1,143 @@
+/*
```

Comment:
Are these two files just copied over from the subscription manager demo? Can't they just be used directly, or symlinked?

Reply:
Yes, we will integrate them in future iterations. For now they are just a copy, if we may need to modify them according to the ota_mqtt callbacks.